domino_data/_feature_store/logging.py

```diff
@@ -0,0 +1,29 @@
+"""Dedicated Feature Store Logging module."""
+
+import os
+import sys
+
+from loguru import logger as _logger
+
+
+def get_feature_store_logger():
+    """Configure and return a logger."""
+    _logger.configure(
+        handlers=[
+            {
+                "format": "[{time}] {message}",
+                "sink": sys.stdout,
+                "enqueue": True,
+            },
+        ],
+        extra={
+            "ip": os.getenv("DOMINO_NODE_IP"),
+            "project": os.getenv("DOMINO_PROJECT_NAME"),
+            "run_id": os.getenv("DOMINO_RUN_ID"),
+            "user": os.getenv("DOMINO_STARTING_USERNAME"),
+        },
+    )
+    return _logger
+
```

Comment:
so the fix here was to reconfigure the logger we were using to create a feature store specific logger? are the changes we needed in the `extra` section?

Reply:
They are actually not being used for now cause in our format we only use time and message. They can be used in the format in the future if we feel they are helpful.

Reply:
hm I guess what I'm trying to figure out is what the change is that made this logger work for what we want

Reply:
This logging.py is copied over from domino_data/logging.py but removing the part that logging to the file which is required by other non-featurestore modules. Job scripts require logging to console so that they can be captured as Job run result. But other non-featurestore module doesn't want this logging. That's why we first implemented as same logging module but with filters. Unfortunately this filter works with other types of logs but not exceptional logs. So I separate the logging for feature store to have a dedicated logging module.